### PR TITLE
last actors

### PR DESCRIPTION
**Pull request by HailSanta**

### CODE REVIEW

File: src/actor_api.c [hunk 1]
```diff
@@ -2,8 +2,8 @@
 #include "effects.h"
 #include "battle/battle.h"
 
-extern s32 D_8029FBD0;
-extern s8 D_8029FBD4;
+extern s32 isGroupHeal;
```
Comment (ethteck) [inline src/actor_api.c:5]: maybe IsGroupHeal since this is a global?

**Review by ethteck — CHANGES_REQUESTED** [review]
kyool

File: src/actor_api.c (hunk 1 above)
Comment (HailSanta) [reply src/actor_api.c:5]: Good catch. Annoyingly, its only used in a single function and has no business being a global. Alas.

Comment (ethteck) [reply src/actor_api.c:5]: this and everything after it in the file can possibly be declared in-function static if we set it up correctly. it'd be good to do a pass of that on the whole game at some point (for symbols only used in one function)

Comment (HailSanta) [reply src/actor_api.c:5]: Agreed, I'd love to see that. Might help us find some "hidden" file splits.

File: src/battle/area/omo2/actor/boss_common.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#ifndef _GENERAL_GUY_COMMON_H_
+#define _GENERAL_GUY_COMMON_H_
+
+#include "mapfs/omo_bt07_shape.h"
+
+enum N(BattleCommon) {
```
Comment (nanaian) [inline src/battle/area/omo2/actor/boss_common.h:6]: `=` alignment is inconsistent here

**Review by nanaian — CHANGES_REQUESTED** [review]

File: src/battle/area/omo2/actor/boss_common.h (hunk 2 above)
Comment (HailSanta) [reply src/battle/area/omo2/actor/boss_common.h:6]: The intent is to add an additional indent level for the enumerated values of each actor var.

Comment (nanaian) [reply src/battle/area/omo2/actor/boss_common.h:6]: ah, I see! 👍

**Review by nanaian — APPROVED** [review]

**Pull request merged**